docs/guides/cookies.md

```diff
@@ -15,6 +15,8 @@ import puppeteer from 'puppeteer';
 
 const browser = await puppeteer.launch();
 
+const context = await browser.createBrowserContext();
```

Comment:
in this example, it needs to be `await browser.cookies()` down below instead of creating a new context.

Reply:
done in 1c34f0c3042dc3060d54ce5d64d6c15298f70d88